charts/umbrella/values.yaml

```diff
@@ -121,11 +128,13 @@ dapsserver:
   persistence:
     accessMode: "ReadWriteOnce"
   omejdn:
-    defaultAdminUser: "admin:ThisiSaSuPERSecretPassword1!11" # unknown
+    defaultAdminUser: *dapsDefaultAdminUser
```

Comment:
Hm, why so complicated? I'd have expected something like this:

```yaml
omejdn:
  # value content should be "user:pass", e.g. "admin:AddThePasswordHere"
  defaultAdminUser: ""
```

To be honest, I did not recognize Line 26. I'd also empty this values like shown above with appropriate comment. Or for simplification: Define the `clientSecret`(Line 26) only once and use `defaultAdminUser: *dapsDefaultAdminUser:*clientSecret` in line 131 and maybe in line 29, if applicable.

Reply:
@carslen
We now use the default value of the `daps` - because otherwise this will always be the fall back value.
If no user/password will be provided it has to be specified (not wanted in e2e-testing).

This secret will also not last long here because daps is deprecated.